app/controllers/api/v1/search_keywords_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchKeywordsController < ApplicationController
+      before_action :authorize!
+
+      def index
+        pagination, keywords = search_keywords_form.search_keywords
+
+        render json: KeywordSerializer.new(keywords, meta: meta_from_pagination(pagination))
+      end
+
+      private
+
+      def search_keywords_form
+        @search_keywords_form ||= SearchKeywordsForm.new(current_user.keywords, search_params, pagination_params)
```

Comment:
Instead of eager load the keyword here, we can query the keyword inside the `KeywordsQuery`

```suggestion
        @search_keywords_form ||= SearchKeywordsForm.new(current_user, search_params, pagination_params)
```